data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,609 @@
+	description "Unfortunately, while Avgi sensors excel in reading basic physical and chemical information, such as the composition of an object, the amount of sensor data required to predict an outfit's functionality means this sensor can be quite slow when scanning a ship's complex systems or cargo."
+
+
+outfit "Diffuse Deflector"
+	category "Systems"
+	cost 1200000
+	thumbnail "outfit/unknown"
+	"mass" 48
+	"outfit space" -48
+	"delayed shield generation" 0.6
+	"delayed shield energy" 1.8
+	"delayed shield heat" 4.8
+	"shield delay" 3
+	description "For its size, you would expect this shield generator to be far more powerful than it actually is. While it is nearly as large as the heaviest human shield generators, it is only about half as strong, while also consuming much more power. Much of this power ends up being shed as waste heat instead of being used to efficiently regenerate a shield."
+
+
+outfit "Specular Deflector"
+	category "Systems"
+	cost 2100000
+	thumbnail "outfit/unknown"
+	"mass" 72
+	"outfit space" -72
+	"delayed shield generation" 1.2
+	"delayed shield energy" 3.6
+	"delayed shield heat" 9.6
+	"shield delay" 3
+	description "The largest shield generator manufactured by the Avgi is impressive, if only because of how far the Avgi have pushed their limited theory of shielding principles. While their engineering ingenuity is admirable, there is a limit to how far you can take an incomplete understanding of the base physics behind shielding, and this generator approaches that limit."
+
+
+outfit "Optical Lasing Generator"
+	category "Systems"
+	cost 120000
+	thumbnail "outfit/unknown"
+	"mass" 30
+	"outfit space" -30
+	"lasing power" 3
+	description "A monolithic slab of semiconductor diode is used to efficiently pump a spool of doped optical fiber. Hundreds of kilometers long, the coiled fibers serve as a lasing medium with the power needed to produce an intense beam of light, and can be gain-switched to pulse at a variety of frequencies. The produced laser light, which can be converted to a number of wavelengths, can be easily tapped and routed via optical fibers or mirrors to a number of turrets on a ship, eliminating the need for the heavy lasing generator to be anywhere near the turret itself."
+	description "The infrared light produced in the coil can either be passed through a frequency doubling crystal to support up to three Green Optical Lasers, or frequency tripled to support a single Blue Optical Laser."
+
+
+
+# Power:
+
+
+outfit "Concentrating Solar Cell"
+	category "Power"
+	cost 12600
+	thumbnail "outfit/unknown"
+	"mass" 1
+	"outfit space" -3
+	"solar collection" 0.3
+	"solar heat" 0.12
+	description "This tiny set of concentrating solar cells efficiently collects sunlight, turning it into usable energy. Thin Fresnel lenses concentrate sunlight onto a small number of highly efficient solar cells, reducing weight and improving efficiency."
+
+
+outfit "Concentrating Solar Array"
```

Comment:
Good enough that it's genuinely useful, but weaker than the Coalition's solar panels. The low mass is interesting, but not a balance issue.